src/scss/_mixins.css

```diff
@@ -0,0 +1,3 @@
+
+
+/*# sourceMappingURL=_mixins.css.map */
```

Comment:
Do we need this file and the _mixins.css.map files?  I'm not clear on what these do and if their essential or not.